sympy/printing/pretty/pretty.py

```diff
@@ -1902,27 +1902,26 @@ def _print_Mul(self, product):
             return prettyForm.__mul__(*a)/prettyForm.__mul__(*b)
 
     # A helper function for _print_Pow to print x**(1/n)
-    def _print_nth_root(self, base, expt):
+    def _print_nth_root(self, base, root):
         bpretty = self._print(base)
 
         # In very simple cases, use a single-char root sign
         if (self._settings['use_unicode_sqrt_char'] and self._use_unicode
-            and expt is S.Half and bpretty.height() == 1
+            and root == 2 and bpretty.height() == 1
```

Comment:
Should this be `root is S.Two`?

Reply:
I think there is no `Two` Attribute for S in SymPy. It is also giving an error regarding the same, so I believe `root == 2` has to be used